Client/Frontend/UIConstants.swift

```diff
@@ -60,6 +60,26 @@ private struct TempStrings {
+    let clearSearchAccessibilityLabel   = NSLocalizedString("Clear Search", tableName: "LoginManager", comment: "Clears the search input field and exits out of input mode")
+    let searchOverlayAccessibilityLabel = NSLocalizedString("Enter Search Mode", tableName: "LoginManager", comment: "Accessibility label for entering search mode for logins")
+    let detailUsernameRowTitle          = NSLocalizedString("username", tableName: "LoginManager", comment: "Title for username row in Login Detail View")
+    let detailPasswordRowTitle          = NSLocalizedString("password", tableName: "LoginManager", comment: "Title for password row in Login Detail View")
+    let detailWebsiteRowTitle           = NSLocalizedString("website", tableName: "LoginManager", comment: "Title for website row in Login Detail View")
+    let deleteLoginDetail               = NSLocalizedString("Delete", tableName: "LoginManager", comment: "Button in login detail screen that deletes the current login")
+    let lastLoginModified               = NSLocalizedString("Last modified %@", tableName: "LoginManager", comment: "Footer label describing when the login was last modifed")
+    let revealPassword                  = NSLocalizedString("Reveal", tableName: "LoginManager", comment: "Reveal password text selection menu item")
+    let openAndFill                     = NSLocalizedString("Open & Fill", tableName: "LoginManager", comment: "Open and Fill website text selection menu item")
+    let deselectAll                     = NSLocalizedString("Deselect All", tableName: "LoginManager", comment: "Title for deselecting all selected logins")
+    let selectAll                       = NSLocalizedString("Select All", tableName: "LoginManager", comment: "Title for selecting all logins")
+    let deletePrompt                    = NSLocalizedString("Are you sure you want to delete all your logins?", tableName: "LoginManager", comment: "Prompt message warning user about deleting all of their logins")
+    let deleteSelectedPrompt            = NSLocalizedString("Are you sure you want to delete the selected logins?", tableName: "LoginManager", comment: "Prompt message warning user about deleting the selected logins")
+    let deleteSinglePrompt              = NSLocalizedString("Are you sure you want to delete this login?", tableName: "LoginManager", comment: "Prompt message warning user about deleting a single login")
+    let deleteSyncedDevices             = NSLocalizedString("Deleted logins will be removed from all your connected devices.", tableName: "LoginManager", comment: "Prompt message warning the user that deleted logins will remove logins from all connected devices")
+    let noResultsFound                  = NSLocalizedString("No Results Found", tableName: "LoginManager", comment: "Title displayed when no results are found after searching")
```

Comment:
+1 for "No logins found" (why is this title case?)

Re: sleroux's — Using 'Deleted' is redundant when using "removed". 
Local:
Are you sure?
Logins will be permanently removed.
[Cancel][Delete]

Synced:
Are you sure?
Logins will be permanently removed from all connected devices.
[Cancel][Delete]